andela/aragon-cfh
# #141404595 Users should be quickly and effectively onboarded onto the app

#### What does this PR do?
Implements an onboarding tour for the Cards for Humanity game.

#### Description of Task to be completed?
As new user I want to see a brief tour of how the game is played and the important controls so that I have sufficient knowledge on how to continue to play the game.

#### How should this be manually tested?
Run the app and open it in a browser.
a) As a new user (or guest), you should be shown a brief explanatory tour of the game when you start or join a game. On ending the tour you should be directed to the gaming screen.
b) As a signed-in user that has seen the tour before, you should not be shown the tour again on starting or joining a game.

#### Any background context you want to provide?
I used Bootstrap Tour to implement the tour and created a route `/api/disabletour` to disable the tour for a given user once seen.

#### What are the relevant pivotal tracker stories?
#141404595

#### Questions:
None for now
